src/components/td/td.jsx

```diff
@@ -77,8 +91,16 @@ Td.propTypes = {
   borderBottom: PropTypes.bool,
   borderLeft: PropTypes.bool,
   align: PropTypes.oneOf(['left', 'right', 'center']),
+  alignScreen: PropTypes.oneOf(['left', 'right', 'center']),
```

Comment:
I am not sure about this `Screen` suffix naming. Does it make sense to use default values for mobile, eg. `align` for mobile and maybe `alignDesktop` for bigger screens?

(This applies to other props named `*Mobile` and `*Screen` as well)